src/_includes/components/cohort-banner.njk

```diff
@@ -0,0 +1,16 @@
+<div>
```

Comment:
@will0101 This was a bit unclear I think. This section of the design is a global footer element, this isn't unique to the VSP pages, so we don't need to add it in this PR. Also, we already have an include for it here https://github.com/ideas42ventures/ideas42ventures.com/blob/main/src/_includes/components/apply-cta.njk

So, let's remove this file `cohort-banner.njk` and in another branch we'll update `apply-cta` with the new text and markup.

Reply:
Removed